src/platform/pret_sdl/sdl2.c

```diff
@@ -1157,6 +1166,8 @@ static inline uint16_t getBgPD(int bgNumber)
     } else if (bgNumber == 3) {
         return REG_BG3PD;
     }
+
+    return 0;
```

Comment:
asserts for checking the BG nums in the beginnings of the functions would be better suited for getBgX, getBgY, getBgPA, getBgPB, getBgPC and getBgPD, instead of (just) the return 0.

I assume it's due to compile warnings, so we can leave the returns in.

Reply:
Yep it's just about compiler warnings